CHANGELOG.md

```diff
@@ -3,6 +3,78 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.29.1
+
+<!-- generated comparing v7.29.0..v7.x -->
+
+_Apr 23, 2025_
+
+We'd like to offer a big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+Following are all team members who have contributed to this release:
+@arminmeh, @cherniavskii, @flaviendelangle, @LukasTy.
```

Comment:
```suggestion
@arminmeh, @cherniavskii, @romgrk, @LukasTy.
```

Reply:
Well, it wasn't mentioned 😅
Perhaps it's a good idea to rerun `pnpm release:changelog` to ensure we don't miss anything.
I usually run it after merging master into the release branch with the `--lastRelease` flag pointing to the commit before the merge, so that it only lists items that have been added by the merge.